prow/config.yaml

```diff
@@ -448,6 +448,110 @@ presubmits:
     rerun_command: "@k8s-bot pull-security-kubernetes-node-e2e test this"
     trigger: "@k8s-bot (pull-security-kubernetes-node-e2e |node e2e )?test this"
 
+  - name: pull-kubernetes-federation-e2e-gce-canary
+    skip_branches:
+    - release-1.4
+    - release-1.5
+    - release-1.6
+    always_run: false
+    skip_report: true
```

Comment:
sorry, i had missed it. added `skip_report: true` to `pull-security-kubernetes-federation-e2e-gce` now